<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VigraQImageViewerBase</class>
 <widget class="QMainWindow" name="VigraQImageViewerBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1269</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VigraQImageViewer</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1269</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_image"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="actionSaveAsPDF"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionZoom_in"/>
    <addaction name="actionZoom_out"/>
    <addaction name="actionReset_zoom"/>
    <addaction name="separator"/>
    <addaction name="actionShow_normalized_image"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDocumentation"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_image">
   <property name="text">
    <string>Load image</string>
   </property>
  </action>
  <action name="actionSave_classified_image">
   <property name="text">
    <string>Save classified image</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="text">
    <string>Zoom &amp;in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="text">
    <string>Zoom &amp;out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionReset_zoom">
   <property name="text">
    <string>&amp;Reset zoom</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionShow_normalized_image">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;normalized image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSet_parameters_and_start_analysis">
   <property name="text">
    <string>Set parameters and start analysis</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionLoad_Recipe">
   <property name="text">
    <string>Load Recipe</string>
   </property>
  </action>
  <action name="actionSave_Recipe">
   <property name="text">
    <string>Save Recipe</string>
   </property>
  </action>
  <action name="actionExport_CSV_SVG">
   <property name="text">
    <string>Export results as CSV &amp;&amp; SVG</string>
   </property>
  </action>
  <action name="actionExport_PDF">
   <property name="text">
    <string>Export view as PDF</string>
   </property>
  </action>
  <action name="actionSaveAsPDF">
   <property name="text">
    <string>Save as PDF</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
